Training.csproj

```diff
@@ -7,4 +7,13 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="a08.Evaluator\**" />

```

Comment:
Repeated mistake of having unnecessary changes with the commit. Kindly correct at least before submitting.